src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -231,12 +253,15 @@ navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
   var resetEnergyExpended = new Uint8Array([1]);
   return characteristic.writeValue(resetEnergyExpended);
 })
-.then(() => {
+.then(_ => {
   console.log('Energy expended has been reset.');
 })
 .catch(error => { console.log(error); });
 {% endhighlight %}
 
+Using a `characteristicvaluechanged` event listener to know when a characteristic
```

Comment:
Wait, is this actually implemented? If so, that's a bug: the spec doesn't say anything about an event being dispatched after writing a value.